src/app/onboard/member/page.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+import React, { useState } from "react";
+import { useRouter } from "next/navigation";
+import { IoArrowBackOutline } from "react-icons/io5";
+
+import { Button } from "@/components/Button";
+import { MembershipTypeButton } from "@/components/MembershipTypeButton";
+
+const MembershipType = () => {
+  const [selectedMembership, setSelectedMembership] = useState<
+    boolean | undefined
+  >(undefined);
```

Comment:
Could you make use of the OnboardingDetailsStore?

Take a look at the onboard/name page to see how it's used.